app/index.tsx

```diff
@@ -81,57 +72,54 @@ const parseDeepLinking = (url: string) => {
 	return null;
 };
 
-export default class Root extends React.Component<{}, IState> {
-	private listenerTimeout!: any;
+const Root = () => {
+	const { width: widthWindow, height: heightWindow, scale: scaleWindow, fontScale: fontScaleWindow } = Dimensions.get('window');
```

Comment:
You're re-running this `Dimensions.get` on every render, which used to be called only on `constructor`.
What you can do is actually call it outside the component.